pkg/cri/server/sandbox_stop.go

```diff
@@ -67,6 +73,10 @@ func (c *criService) stopPodSandbox(ctx context.Context, sandbox sandboxstore.Sa
 		if err := c.stopContainer(ctx, container, 0); err != nil {
 			return fmt.Errorf("failed to stop container %q: %w", container.ID, err)
 		}
+
+		span.AddEvent("stopped container in sandbox",
```

Comment:
same here we should do the events once .. and probably inside the inner func after conversion to the full id

Reply:
ack